CoreInteractiveGraph/test/unit/src/au/gov/asd/tac/constellation/graph/interaction/gui/VisualGraphOpenerNGTest.java

```diff
@@ -47,35 +51,82 @@ public void testOpenGraph() {
         final FileObject mockFileObject = mock(FileObject.class);
         final File mockFile = mock(File.class);
         final String path = "mocked path";
+        final Path mockPath = mock(Path.class);
         final Long lastModified = 123L;
 
         when(mockGdo.getPrimaryFile()).thenReturn(mockFileObject);
         when(mockFileObject.getPath()).thenReturn(path);
-        when(mockFile.getPath()).thenReturn(path);
+        when(mockFile.getAbsolutePath()).thenReturn(path);
         when(mockFile.lastModified()).thenReturn(lastModified);
 
         // Check mocks work
         assertEquals(mockGdo.getPrimaryFile(), mockFileObject);
-        assertEquals(mockFile.getPath(), path);
+        assertEquals(mockFile.getAbsolutePath(), path);
         assertEquals((Long) mockFile.lastModified(), lastModified);
 
-        try (MockedStatic<FileUtil> mockFileUtil = Mockito.mockStatic(FileUtil.class, Mockito.CALLS_REAL_METHODS)) {
+        try (MockedStatic<FileUtil> mockFileUtil = Mockito.mockStatic(FileUtil.class, Mockito.CALLS_REAL_METHODS); MockedStatic<Paths> mockPaths = Mockito.mockStatic(Paths.class, Mockito.CALLS_REAL_METHODS)) {
             // Set up FileUtil mock
             mockFileUtil.when(() -> FileUtil.toFile(mockFileObject)).thenReturn(mockFile);
             assertEquals(FileUtil.toFile(mockFileObject), mockFile);
+            // Set up Paths mock
+            mockPaths.when(() -> Paths.get(path)).thenReturn(mockPath);
+            assertEquals(Paths.get(path), mockPath);
+
+            final VisualGraphOpener instance = new VisualGraphOpener();
+            instance.openGraph(mockGdo);
+
+            // Assert that the path was added to the list, 
+            // The list shouldn't be empty at the moment
+            assertFalse(VisualGraphOpener.getOpeningGraphs().isEmpty());
+            assertTrue(VisualGraphOpener.getOpeningGraphs().contains(mockPath));
+
+            // Open again
+            instance.openGraph(mockGdo);
+            // Assert path still in list
+            assertFalse(VisualGraphOpener.getOpeningGraphs().isEmpty());
+            assertTrue(VisualGraphOpener.getOpeningGraphs().contains(mockPath));
+        }
+
+        System.clearProperty("java.awt.headless");
+    }
+
+    /**
+     * Test of openGraph method, of class VisualGraphOpener.
+     */
+    @Test
+    public void testOpenGraphDummyFile() {
+        System.out.println("openGraphDummyFile");
+        System.setProperty("java.awt.headless", "true");
```

Comment:
If you are setting this to true at the start of all the tests and back to false at the end, then you could put these in the setUpClass() and tearDownClass() functions.